internal/query/projection/user_auth_method.go

```diff
@@ -47,9 +49,10 @@ func newUserAuthMethodProjection(ctx context.Context, config crdb.StatementHandl
 			crdb.NewColumn(UserAuthMethodResourceOwnerCol, crdb.ColumnTypeText),
 			crdb.NewColumn(UserAuthMethodInstanceIDCol, crdb.ColumnTypeText),
 			crdb.NewColumn(UserAuthMethodNameCol, crdb.ColumnTypeText),
+			crdb.NewColumn(UserAuthMethodOwnerRemovedCol, crdb.ColumnTypeBool, crdb.Default(false)),
 		},
 			crdb.NewPrimaryKey(UserAuthMethodInstanceIDCol, UserAuthMethodUserIDCol, UserAuthMethodTypeCol, UserAuthMethodTokenIDCol),
-			crdb.WithIndex(crdb.NewIndex("auth_meth_ro_idx", []string{UserAuthMethodResourceOwnerCol})),
+			crdb.WithIndex(crdb.NewIndex("auth_meth4_1_ro_idx", []string{UserAuthMethodResourceOwnerCol})),
```

Comment:
is this name correct 4_1?

Reply:
Corrected it, also changed all existing indexes to similar naming with the count at the end of the name